packages/@webex/plugin-cc/src/cc.ts

```diff
@@ -115,8 +115,8 @@ export default class ContactCenter extends WebexPlugin implements IContactCenter
         })
         .then(async (data: WelcomeEvent) => {
           const agentId = data.agentId;
-          const agentConfig = new AgentConfig(agentId, this.$webex, this.httpRequest);
-          this.agentConfig = await agentConfig.getAgentProfile();
+          const orgId = this.$webex.credentials.getOrgId();
```

Comment:
Have we been able to fix th broken test arising from this line?

Reply:
Yes. i parameterised the orgid for services. thought its a better approach as anyone can use this services layer by passing orgID.